packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3067,7 +3242,10 @@ class Flex extends MultiChildRenderObjectWidget {
 /// new Row(
 ///   children: <Widget>[
 ///     const FlutterLogo(),
-///     const Text('Flutter\'s hot reload helps you quickly and easily experiment, build UIs, add features, and fix bug faster. Experience sub-second reload times, without losing state, on emulators, simulators, and hardware for iOS and Android.'),
+///     const Text("Flutter's hot reload helps you quickly and easily experiment, "
+///         'build UIs, add features, and fix bug faster. Experience sub-second '
+///         'reload times, without losing state, on emulators, simulators, and '
+///         'hardware for iOS and Android.'),
```

Comment:
The lack of wrapping here was very intentional. It drives home the point that the string is really long:
https://master-docs-flutter-io.firebaseapp.com/flutter/widgets/Row-class.html

Reply:
Heh.  OK, I just thought it was pretty hard to read... :-) (Reverted)